docker-compose.yml

```diff
@@ -232,7 +232,7 @@ services:
 
   notification:
     build: notification-services/0.0.1-3.2
-    image: oapass/notification-services:0.0.1-3.2-SNAPSHOT@sha256:1d4647d0c5a589e9f11df06be26b01d957698af20b91c12bea9c768b9c44b8cb
+    image: oapass/notification-services:0.0.1-3.2-SNAPSHOT@sha256:5fb17280141daaa0f794f0b23829913708f7b4ab463a81588f0af7b14f99f297
```

Comment:
Curious as to why the image needs updating?

Reply:
The notification.json config changed, and `COPY notification.json /notification.json`.  The one in `pass-docker` didn't seem to be bind mounted as a volume, therefore I concluded it was baked into the image